webapp/client/src/components/AnchorButton.test.js

```diff
@@ -1,15 +1,15 @@
 import React from "react";
 import { render, screen } from "@testing-library/react";
-import DownloadLink from "./DownloadLink";
+import AnchorButton from "./AnchorButton";
 
 const MOCK_PROPS = {
   text: "anchor text",
   url: "/some/link/path",
 };
 
-describe("DownloadLink", () => {
+describe("AnchorButton", () => {
   it("should render the text", () => {
-    render(<DownloadLink {...MOCK_PROPS} />);
+    render(<AnchorButton {...MOCK_PROPS} />);
 
     expect(screen.getByText(MOCK_PROPS.text)).toBeInTheDocument();
     expect(screen.getByText(MOCK_PROPS.text).closest("a")).toHaveAttribute(
```

Comment:
Shouldn't we check at least 2 cases  (with and without attribute download)?

Reply:
Done! :D thanks, totally forgotten.